hosted/eks/p1/p1_provisioning_test.go

```diff
@@ -76,7 +76,36 @@ var _ = Describe("P1Provisioning", func() {
 			Eventually(func() bool {
 				cluster, err := ctx.RancherAdminClient.Management.Cluster.ByID(cluster.ID)
 				Expect(err).To(BeNil())
-				return cluster.Transitioning == "error" && strings.Contains(cluster.TransitioningMessage, "node group names must be unique")
+				return cluster.Transitioning == "error" && strings.Contains(cluster.TransitioningMessage, "is not unique within the cluster")
+			}, "1m", "3s").Should(BeTrue())
+		})
+
+		It("Fail to create cluster with different k8s versions on control plane and on nodegroup", func() {
+			testCaseID = 127
+
+			k8sVersions, err := helper.ListEKSAllVersions(ctx.RancherAdminClient)
+			Expect(err).To(BeNil())
+
+			cpK8sVersion := k8sVersions[0]
+			ngK8sVersion := k8sVersions[1]
```

Comment:
```suggestion
			cpK8sVersion := k8sVersions[1]
			ngK8sVersion := k8sVersions[0]
```
To test exact case "CP as vXX and NG as vXX+1"

Reply:
Good catch, I tried both variants earlier in CI and there was no difference in returned error so it should be ok.